style.css

```diff
@@ -0,0 +1,397 @@
+* {
+	box-sizing:border-box;
+	scroll-behavior: smooth;
+}
+
+.nav-link {
+	display: flex;
+	justify-content: flex-start;
+	gap: 20px;
+	align-items: center;
+}
+
+img[alt="french"] {
+	max-width:30px;
+	border:1px solid white;
+	border-radius: 15px;
+}
+
+img[alt="UK"] {
+	max-width:30px;
+	border:1px solid white;
+	border-radius: 15px;
+}
```

Comment:
instead of selecting by alt, images should have a class that describes what they do on the page (as discussed)